### PR TITLE
docs: table shows the support status of built-in plugins

### PR DESCRIPTION
**Pull request by SyMind**
## Summary

Table shows the support status of Rspack for the built-in plugins in Webpack.

## Checklist

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -160,7 +160,6 @@ jobs:
     name: Lint and format code
     runs-on: ubuntu-latest
     needs: [check-changed]
-    if: ${{ needs.check-changed.outputs.changed == 'true' }}
```
Comment (chenjiahan) [inline .github/workflows/ci.yml:163]: why is this removed

Comment (SyMind) [reply .github/workflows/ci.yml:163]: I'd like to proceed with the website CI, but it seems this change shouldn't be made, I'll revert it first.

File: website/docs/zh/plugins/webpack/index.mdx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { Table } from '@builtIns';
+import { PluginSupportStatusTable } from '../../../../components/PluginSupportStatusTable';
+
+# 同步自 webpack 的内置插件
+
+以下表格展示了 Rspack 相对于 Webpack 内置插件的支持情况。对于尚未实现的插件或功能，如果你有兴趣参与开发，我们非常欢迎你来参与。
```
Comment (fi3ework) [inline website/docs/zh/plugins/webpack/index.mdx:6]: Nitpick: webpack's name is in lower-case.

```suggestion
以下表格展示了 Rspack 相对于 webpack 内置插件的支持情况。对于尚未实现的插件或功能，如果你有兴趣参与开发，我们非常欢迎你来参与。
```

File: website/docs/en/plugins/webpack/index.mdx [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { Table } from '@builtIns';
+import { PluginSupportStatusTable } from '../../../../components/PluginSupportStatusTable';
+
+# Synchronized Built-in Plugins from webpack
+
+The table below shows the support status of Rspack for the built-in plugins in Webpack. If you are interested in participating in the development of plugins or features that have not yet been implemented, we would be very happy to have you join us.
```
Comment (chenjiahan) [inline website/docs/en/plugins/webpack/index.mdx:6]:
```suggestion
The table below shows the support status of Rspack for the built-in plugins in webpack. If you are interested in participating in the development of plugins or features that have not yet been implemented, we would be very happy to have you join us.
```
